lib/index.js

```diff
@@ -18,6 +18,9 @@ function new_asset_id() {
 class ValidationError extends Error {
 }
 
+class InvalidArgumentError extends Error {
```

Comment:
Ugh - originally I made no name specified to throw this error and later swapped to no error but didn't remove this.